src/benchmarks/reconstruction/femc_studies/femc_studiesProcessor.cc

```diff
@@ -482,15 +481,15 @@ void femc_studiesProcessor::Process(const std::shared_ptr<const JEvent>& event)
     m_log->info("-----> found {} clusters" , clusters_calo.size());
     hRecNClusters_E_eta->Fill(mcenergy, clusters_calo.size(), mceta);
     int iCl = 0;
-    for (auto& cluster : clusters_calo) {
+    for (const auto cluster : clusters_calo) {
```

Comment:
Applies here.

Reply:
But the PODIO collection iterators don't return a reference https://github.com/AIDASoft/podio/blob/babec745c12727cd5da590abbe1e3cf1ab8743ad/python/templates/macros/iterator.jinja2#L33-L36

Reply:
Right. I keep thinking they are more std::vector than they actually are.